pkgs/applications/networking/remote/chrome-remote-desktop/default.nix

```
@@ -0,0 +1,104 @@
+{ enableNewSession ? false, stdenv, pkgs, lib, config, fetchurl, fetchgit, dpkg, python3, glibc, glib, pam, nss
+, nspr, expat, gtk3, dconf, xorg, fontconfig, dbus, alsa-lib, shadow, mesa, libdrm, libxkbcommon, wayland }:
+stdenv.mkDerivation rec {
+  pname = "chrome-remote-desktop";
+  version = "unstable-2022-02-03";
+
+  src = fetchurl {
+    sha256 = "sha256-CNE3kvAj7Jp4cB5x2D/YUA1H9Ri397C6rxQBRmstz1c=";
+    url = "https://dl.google.com/linux/direct/chrome-remote-desktop_current_amd64.deb";
+  };
+
+  buildInputs = [ pkgs.makeWrapper ];
+
+  dontBuild = true;
+  dontConfigure = true;
+
+  unpackPhase = ''
+    ${dpkg}/bin/dpkg -x $src $out
+  '';
+
+  installPhase = ''
+    mkdir $out/bin
+    makeWrapper $out/opt/google/chrome-remote-desktop/chrome-remote-desktop $out/bin/chrome-remote-desktop
+  '';
+
+  replacePrefix = "/opt/google/chrome-remote-desktop";
+  replaceTarget = "/run/current-system/sw/bin/./././";
+
+  patchPhase =
+  ''
+    sed \
+    -e '/^.*sudo_command =/ s/"gksudo .*"/"pkexec"/' \
+    -e '/^.*command =/ s/s -- sh -c/s sh -c/' \
+    -i $out/opt/google/chrome-remote-desktop/chrome-remote-desktop
+    substituteInPlace $out/etc/opt/chrome/native-messaging-hosts/com.google.chrome.remote_desktop.json --replace $replacePrefix/native-messaging-host $out/$replacePrefix/native-messaging-host
+    substituteInPlace $out/etc/opt/chrome/native-messaging-hosts/com.google.chrome.remote_assistance.json --replace $replacePrefix/remote-assistance-host $out/$replacePrefix/remote-assistance-host
+
+    substituteInPlace $out/$replacePrefix/chrome-remote-desktop --replace "USER_SESSION_PATH = " "USER_SESSION_PATH = \"/run/wrappers/bin/crd-user-session\" #"
+    substituteInPlace $out/$replacePrefix/chrome-remote-desktop --replace /usr/bin/python3 ${python3.withPackages (ps: with ps; [ psutil ])}/bin/python3
```

substituteInPlace can take multiple replace arguemnts.

```suggestion
    substituteInPlace $out/$replacePrefix/chrome-remote-desktop \
      --replace "USER_SESSION_PATH = " "USER_SESSION_PATH = \"/run/wrappers/bin/crd-user-session\" #" \
      --replace /usr/bin/python3 ${python3.withPackages (ps: with ps; [ psutil ])}/bin/python3
```
and so on